### ISSUE TITLE
Does not work on Linux Wayland

### ISSUE THREAD

**Issue by Algram** [hyperion-project/hyperion-obs-plugin#6]
Hello,

I am getting the following error when starting the plugin:

```
Connected to localhost, port: 19400
Reply received with error: Size of image data does not match with the width and height
Reply received with error: Size of image data does not match with the width and height
Reply received with error: Size of image data does not match with the width and height
Reply received with error: Size of image data does not match with the width and height
Connection to Hyperion server closed
```

Is there any way I could debug this further?

Thanks!

**Comment by Paulchen-Panther** [comment]
Which Hyperion version?

**Comment by Algram** [comment]
@Paulchen-Panther 2.0.12, so the latest version

**Comment by Algram** [comment]
I just tried v1.0.0 of the plugin and that works. So a change in between that is responsible for it breaking.

**Comment by Paulchen-Panther** [comment]
I was just about to write you that you should please try it with version 1.0. The changes in version 1.0.1 are in the master of hyperion.ng. Since no new version of hyperion.ng has been released yet, version 1.0.1 only works in the current master.
Thanks for the issue. I will add a note at the 1.0.1 release.
